ci/gpu/build.sh

```diff
@@ -83,7 +83,6 @@ gpuci_conda_retry install -y \
 
 # https://docs.rapids.ai/maintainers/depmgmt/
 # gpuci_conda_retry remove --force rapids-build-env rapids-notebook-env
-gpuci_conda_retry install -y "numpy<1.20"
```

Comment:
There used to be a different text here indicating how to install a package that we should add back in.

Reply:
fixed